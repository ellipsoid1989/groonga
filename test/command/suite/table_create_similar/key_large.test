table_create Users TABLE_HASH_KEY|KEY_LARGE ShortText

table_create_similar UsersSimilar Users

dump
