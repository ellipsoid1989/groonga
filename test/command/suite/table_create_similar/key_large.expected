table_create Users TABLE_HASH_KEY|KEY_LARGE ShortText
[[0,0.0,0.0],true]
table_create_similar UsersSimilar Users
[[0,0.0,0.0],true]
dump
table_create Users TABLE_HASH_KEY|KEY_LARGE ShortText

table_create UsersSimilar TABLE_HASH_KEY|KEY_LARGE ShortText
